modules/thermal_hydraulics/src/closures/Closures1PhaseTHM.C

```diff
@@ -0,0 +1,379 @@
+      {
+        params.set<MooseEnum>("bundle_array") = "SQUARE";
+      }
+      else if (flow_channel.getHeatTransferGeometry() == 2)
+      {
+        params.set<MooseEnum>("bundle_array") = "TRIANGULAR";
+      }
+      _sim.addMaterial(class_name, genName(flow_channel.name(), "whtc_mat", i), params);
+      break;
+    }
+    case WallHTCClosureType::LYON:
+    {
+      const std::string class_name = "ADWallHeatTransferCoefficientLyonMaterial";
+      InputParameters params = _factory.getValidParams(class_name);
+      params.set<MaterialPropertyName>("Hw") = flow_channel.getWallHTCNames1Phase()[i];
+      params.set<MaterialPropertyName>("rho") = FlowModelSinglePhase::DENSITY;
+      params.set<MaterialPropertyName>("vel") = FlowModelSinglePhase::VELOCITY;
+      params.set<MaterialPropertyName>("D_h") = FlowModelSinglePhase::HYDRAULIC_DIAMETER;
+      params.set<MaterialPropertyName>("cp") =
+          FlowModelSinglePhase::SPECIFIC_HEAT_CONSTANT_PRESSURE;
+      params.set<MaterialPropertyName>("mu") = FlowModelSinglePhase::DYNAMIC_VISCOSITY;
+      params.set<MaterialPropertyName>("k") = FlowModelSinglePhase::THERMAL_CONDUCTIVITY;
+      params.set<MaterialPropertyName>("T") = FlowModelSinglePhase::TEMPERATURE;
+      params.set<MaterialPropertyName>("T_wall") = flow_channel.getWallTemperatureNames()[i];
+      params.set<std::vector<SubdomainName>>("block") = flow_channel.getSubdomainNames();
+      _sim.addMaterial(class_name, genName(flow_channel.name(), "whtc_mat", i), params);
+      break;
+    }
+    case WallHTCClosureType::KAZIMI_CARELLI:
+    {
+
+      const std::string class_name = "ADWallHeatTransferCoefficientKazimiMaterial";
+      InputParameters params = _factory.getValidParams(class_name);
+      params.set<MaterialPropertyName>("Hw") = flow_channel.getWallHTCNames1Phase()[i];
+      params.set<MaterialPropertyName>("rho") = FlowModelSinglePhase::DENSITY;
+      params.set<MaterialPropertyName>("vel") = FlowModelSinglePhase::VELOCITY;
+      params.set<MaterialPropertyName>("D_h") = FlowModelSinglePhase::HYDRAULIC_DIAMETER;
+      params.set<MaterialPropertyName>("cp") =
+          FlowModelSinglePhase::SPECIFIC_HEAT_CONSTANT_PRESSURE;
+      params.set<MaterialPropertyName>("mu") = FlowModelSinglePhase::DYNAMIC_VISCOSITY;
+      params.set<MaterialPropertyName>("k") = FlowModelSinglePhase::THERMAL_CONDUCTIVITY;
+      params.set<MaterialPropertyName>("T") = FlowModelSinglePhase::TEMPERATURE;
+      params.set<MaterialPropertyName>("T_wall") = flow_channel.getWallTemperatureNames()[i];
+      params.set<std::vector<SubdomainName>>("block") = flow_channel.getSubdomainNames();
+      params.set<Real>("PoD") = flow_channel.getParam<Real>("PoD");
+      if (flow_channel.getParam<Real>("PoD") == 1.0)
+      {
+        mooseDoOnce(mooseWarning(
+            "You are using a rod bundle correlation with the default Pitch-to-Diameter "
+            "ratio value, P/D=1.0, to change this do: GlobalParams/PoD='new value'"));
```

Comment:
Same comment as above. Change the other occurences

Reply:
OK

Reply:
done